pkg/controller/acicontainersoperator.go

```diff
@@ -401,6 +518,22 @@ func (c *Controller) ReadConfigMap(field string) ([]byte, error) {
 	return raw, err
 }
 
+func (c *Controller) WriteConfigMap(field string, data *corev1.ConfigMap) (error) {
+	rawIn := json.RawMessage(data.Data["spec"])
+	data_byte,err := rawIn.MarshalJSON()
+	if err != nil{
+		log.Error(err)
+		return err
+	}
+	err = ioutil.WriteFile( field, data_byte, 0777)
+	if err != nil {
+		log.Error(err)
+		return err
+	}
+	log.Debug("Config-Map is:  ", (string(data_byte)))
```

Comment:
I think we should put an info level log statement stating that we are writing the config-map (name?). The actual contents of the map can be at debug level, but if the config is base-encoded perhaps there is no need to log it.

Reply:
Yes the config-map is big base encoded log.  I will change the debug to info level with config map name